### ISSUE TITLE
Vote Notifications

### ISSUE THREAD

**Issue by kharoof** [feruzm/esteem#44]
The notifications feature works great and is one of my favourite aspects to the app. You get notifications when people mention you, comment, follow etc which is very useful. 

The voting notification isn't that useful however as you get too many when you post something new so most people probably have it turned off. What might be interesting would be to have an option where you got notified on posts greater than 1 day old that got votes. Or have different time limits that the user could select.

**Comment by feruzm** [comment]
you are right it is mostly for new users and people who don't want to miss any voter 😄 

Notification in itself can be very complex setting, there will be `subscription` feature later on which gives complimentary set of features where you can subscribe to post, user, blog, etc.
